todo-mvc/tests/intern.ts

```diff
@@ -55,16 +55,7 @@ export const loaderOptions = {
 		{ name: 'tests', location: '_build/tests' },
 		{ name: 'chai', location: 'node_modules/chai', main: 'chai' },
 		{ name: 'dojo', location: 'node_modules/intern/node_modules/dojo' },
-		{ name: 'dojo-actions', location: 'node_modules/dojo-actions' },
-		{ name: 'dojo-app', location: 'node_modules/dojo-app' },
-		{ name: 'dojo-compose', location: 'node_modules/dojo-compose' },
-		{ name: 'dojo-core', location: 'node_modules/dojo-core' },
-		{ name: 'dojo-dom', location: 'node_modules/dojo-dom' },
-		{ name: 'dojo-has', location: 'node_modules/dojo-has' },
-		{ name: 'dojo-routing', location: 'node_modules/dojo-routing' },
-		{ name: 'dojo-shim', location: 'node_modules/dojo-shim' },
-		{ name: 'dojo-stores', location: 'node_modules/dojo-stores' },
-		{ name: 'dojo-widgets', location: 'node_modules/dojo-widgets' },
+		{ name: '@dojo', location: 'node_modules/@dojo' },
```

Comment:
Same question, didn't realize or remember that we could then just do @dojo... if so, very cool, I should update the blog post about it.